backend/src/controllers/chats.js

```diff
@@ -16,8 +15,8 @@ exports.list = async (ctx) => {
 exports.create = async (ctx) => {
   const params = ctx.request.body;
 
-  const chat = await database.Chat.create({message: params.message});
+  const chat = await database.Chat.create({ message: params.message });
 
-  ctx.body = await chat.toJSON();
+  ctx.body = chat;
```

Comment:
this looks suspicious as a read code (chat is an object and setting it to body might be misleading to a reader)

Reply:
I think it makes sense that the chat created object is returned in the response body